pbrd/pbr_vty.c

```diff
@@ -317,7 +317,7 @@ DEFPY (pbr_table_range,
 
 DEFPY (pbr_rule_range,
 	pbr_rule_range_cmd,
-	"[no] pbr rule range (300-1300)$start (400-1400)$end",
+	"[no] pbr rule range (300-900)$start (400-1000)$end",
 	NO_STR
```

Comment:
should not it be 999 instead of 1000 ?

Reply:
This is removed in a future commit that I will upstream after this PBRD branch gets in.

Reply:
ok